index.html

```diff
@@ -15,14 +15,254 @@
   <link href="https://fonts.googleapis.com/css?family=Gloria+Hallelujah|Varela+Round" rel="stylesheet">
   <!-- main css -->
   <link rel="stylesheet" type="text/css" href="css/style.css" />
-  <!-- Font -->
-  <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
   <!-- Animate CSS -->
   <link rel="stylesheet" href="css/animate.css">
-
 </head>
   <body>
+    <!-- START NAV AND BANNER FEAT#12, BUILD BY ZUBAIR -->
+    <!-- start feat: nav -->
+    <nav class="navbar navbar-expand-lg sticky-top nav-header">
```

Comment:
Navbar should collapse on medium size screen.
```suggestion
    <nav class="navbar navbar-expand-md sticky-top nav-header">
```